pkg/rtc/participant.go

```diff
@@ -238,6 +238,14 @@ func (p *ParticipantImpl) GetClientConfiguration() *livekit.ClientConfiguration
 	return p.params.ClientConf
 }
 
+func (p *ParticipantImpl) ICEConnectionType() types.ICEConnectionType {
+	transport := p.TransportManager.getTransport(true)
+	if transport != nil {
+		return transport.ICEConnectionType()
+	}
+	return types.ICEConnectionTypeUnknown
+}
```

Comment:
In light of recent changes, would like to propose this call something like `return p.TransportManager.PrimaryTransportIceConnectionType()` and move the implementation inside transport manager. That way `getTransport` can remain a private method.

Reply:
yup, that makes sense. will update.